package.json

```diff
@@ -2,27 +2,29 @@
   "name": "tablesort",
   "description": "A sorting component for HTML tables",
   "version": "4.1.0",
-  "author": "tristen",
+  "author": {
+    "name": "tristen",
+    "url": "http://tristen.ca"
+  },
   "license": "MIT",
   "ender": "./ender.js",
   "main": "./src/tablesort.js",
   "homepage": "http://tristen.ca/tablesort/demo/",
   "scripts": {
-    "build": "grunt",
+    "build": "{ preamble & uglifyjs src/tablesort.js -c -m; } > tablesort.min.js",
```

Comment:
Yeah... well I wonder if there's another option then. Grunt is just being used to append a banner to the script. It could be so much simpler.